README.adoc

```diff
@@ -20,6 +20,7 @@ https://fmi-standard.org/downloads/[Releases] and the https://fmi-standard.org/d
 == Branches
 
 - `master` -- development of the next release
+Note: During development of the next FMI version the XSD files for the `modeldescription.xml` file are to be considered normative compared to the standard text in case of inconsistencies except for comments, where the standard text is to be considered normative. Please open issues and/or PRs in this case of discovering inconsistencies.
```

Comment:
IMHO the text, XSDs and header files should be normative. Since we only include text from the headers in the AsciiDoc files, I don't see how they can be inconsistent.

Reply:
The XSD files can differ from the standard - can it not?

Reply:
Are all the pictures of the schema elements in the standard autogenerated from the XSD? Otherwise I see the standard potentially deviating during development.
But I do not mind removing this and just keeping the text on fmiVersion

Reply:
> Otherwise I see the standard potentially deviating during development.

If there are any inconsistencies (e.g. outdated screenshots) it's an issue that needs to be resolved.